templates/v2/analysis/report_template.html

```diff
@@ -1,43 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-    <meta charset="utf-8">
-    <title>{{ title }}</title>
-</head>
+    <head>
+        <meta charset="utf-8">
+        <title>{{ title }}</title>
+    </head>
 
-<body>
-    <div>
+    <body>
         <div>
-            <h1>{{ title }}</h1>
             <div>
-                <p>The below analysis shows the number of people who had a code recorded in their primary care record from <a href="{{ codelist_1_link }}"> this <strong>{{ codelist_1_name }}</strong>
-                    codelist</a> each month in the period between {{ start_date }} and {{ end_date }} who also had a code recorded from <a href="{{ codelist_2_link }}"> this codelist for
-                    <strong>{{ codelist_2_name }}</strong></a> in the same month or up to <strong>{{ time_value }} {{ time_scale }} {{ time_event }}</strong> the last day of the month.
-                    {% if breakdowns|length >0 %}
-                    A breakdown of this measure by
-                    {% for b in breakdowns %}
-                    <span>
-                        <strong>{{ b.title|lower }}</strong>
-                        {% if not loop.last %},{% endif %}
-                    </span>
-                    {% endfor %} is also provided.
-                    {% endif %}
+                <h1>{{ title }}</h1>
+                <div>
+                    <p>The below analysis shows the number of people who had a
+                        code recorded in their primary care record from <a
+                            href="{{ codelist_1_link }}"> this <strong>{{
+                                codelist_1_name }}</strong>
+                            codelist</a> each month in the period between {{
+                        start_date }} and {{ end_date }} who also had a code
+                        recorded from <a href="{{ codelist_2_link }}"> this
+                            codelist for
+                            <strong>{{ codelist_2_name }}</strong></a> in the
+                        same month or up to <strong>{{ time_value }} {{
+                            time_scale }} {{ time_event }}</strong> the last day
```

Comment:
Have you tested that the templating still works with the variable interpolations split over newlines like this?

Reply:
I was curious, so I tested it locally using smush.py and the templating does seem to work okay.

Reply:
It all looks ok when run locally